charts/kong/example-values/minimal-kong-enterprise-k8s.yaml

```diff
@@ -0,0 +1,18 @@
+
+
+image:
+  repository: kong-docker-kong-enterprise-k8s.bintray.io/kong-enterprise-k8s
+  tag: 2.0.4.1-centos
+  pullSecrets:
+    - you-must-create-a-registry-secret
```

Comment:
Let's rename this to the standard secret names that we use in our documentation.
Why?
1. Makes it ready to use; most enterprise users follow the guide.
2. We can include a comment if needed.
3. The ultimate goal here would be to then include a link to this file in the `helm` install section of [this](https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/deployment/k4k8s-enterprise.md#helm) doc. The current link in that doc is out of date.

This comment applies to all the files in this PR.